iroh/examples/transfer.rs

```diff
@@ -32,6 +33,10 @@ enum Commands {
         relay_url: Option<String>,
         #[clap(long, default_value = "false")]
         relay_only: bool,
+        #[clap(long)]
+        pkarr_relay_url: Option<String>,
+        #[clap(long)]
+        dns_url: Option<String>,
```

Comment:
Having this named `dns_url` is very confusing.  I had to look up the `DnsDiscovery` docs to understand what was going on.  Why not call this `dns_origin_domain`?  (And of course everywhere else too)